pandas-stubs/core/series.pyi

```diff
@@ -1982,26 +1977,30 @@ class Series(IndexOpsMixin[S1], NDFrame):
     @final
     def ewm(
         self,
-        com: float | None = ...,
-        span: float | None = ...,
-        halflife: float | None = ...,
-        alpha: float | None = ...,
+        com: float | None = None,
+        span: float | None = None,
+        halflife: float | None = None,
+        alpha: float | None = None,
         min_periods: int = 0,
         adjust: _bool = True,
         ignore_na: _bool = False,
+        axis: Axes = ...,
```

Comment:
```suggestion
        axis: Axes = 0,
```

Reply:
Oh interesting, the annotation is wrong here, `Axes` should be `Axis`